lib/validator.js

```diff
@@ -47,6 +47,8 @@ class Validator {
 		if (opts) {
 			deepExtend(this.opts, opts);
 			if (opts.defaults) deepExtend(this.defaults, opts.defaults);
+			// eslint-disable-next-line no-console
+			if (!this.opts.useNewCustomCheckerFunction) console.warn("DeprecationWarning: set 'useNewCustomCheckerFunction = true' in FastestValidator constructor"); // todo: add link
```

Comment:
This mode is very aggressive. It means, all existing users will get this warning message, either don't use custom rules. And in Moleculer you can't add Validator options easily, so they will always get this warning. 
I think no need this warning because old-style will work in v1.x version. After v2.x will be this the default but we will denote It in the changelog. So plz, remove this warning.

Reply:
Users can use old fastest-validator versions without worry! They will only see this message when they have updated the fastest-validator version, I've seen this pattern in many other packages, like mongodb (native driver) - express, etc.. , it's pretty common.

I have no problem with removing this, but a trigger is needed to that users update their codes

Reply:
You can use current fastest-validator version(1.3.0) for Moleculer until it reaches version 2.

Reply:
Moleculer uses `^1.0.0` so the validator can be updated automatically. So my problem is that in Moleculer you can't set this option easily.